packages/atomic/src/components/result-lists/grid-display-results.tsx

```diff
@@ -12,14 +12,16 @@ export const GridDisplayResults: FunctionalComponent<ResultsProps> = (
     });
 
     return (
-      <LinkWithResultAnalytics
-        part="result-list-grid-clickable"
-        onSelect={() => interactiveResult.select()}
-        onBeginDelayedSelect={() => interactiveResult.beginDelayedSelect()}
-        onCancelPendingSelect={() => interactiveResult.cancelPendingSelect()}
-        href={props.resultListCommon.getUnfoldedResult(result).clickUri}
-        target="_self"
-      >
+      <div part="result-list-grid-parent">
```

Comment:
Suggestion: 
```suggestion
      <div part="result-list-grid-clickable-container">
```

Reply:
hmm it contains the clickable link & the result too? maybe `result-list-grid-container`